var/spack/repos/builtin/packages/fujitsu-ssl2/package.py

```diff
@@ -142,3 +142,12 @@ def setup_dependent_build_environment(self, env, dependent_spec):
         env.append_flags(
             "FCC_ENV", "-idirafter " + path
         )
+
+    @property
+    def headers(self):
+        path = join_path(
+            self.spec.prefix, "clang-comp", "lib64", "clang", "7.1.0", "include"
```

Comment:
Is it always `lib64` or will it be `lib` on some systems? Is it always this same version of clang? We could just search recursively from the beginning of the prefix. Or we could use wildcards.

Reply:
Thank you for your advice.
I fixed the recipe to do not include "version number" and "lib64" in path according to your advice .